test/plugin/test_in_forward.rb

```diff
@@ -7,6 +7,22 @@
 require 'fluent/plugin/in_forward'
 
 class ForwardInputTest < Test::Unit::TestCase
+  class << self
+    def startup
+      if Fluent.windows?
+        socket_manager_path = ServerEngine::SocketManager::Server.generate_path
+      else
+        socket_manager_path = ServerEngine::SocketManager::Server.generate_path + __method__.to_s
+      end
```

Comment:
serverengine's API generates
・in unix
generates Unix domain socket and path is time + pid
・in windows
generates TCPServer and path is unused port.

Then, in unix path will be same when the api is called at same time and in same process.
(In windows, it will be not same bacause it will find in each time when it's called)
So I added ``` __method__.to_s``` not to be same in unix.

But It seems to be confusing and I realized we need to create only one socket manager server before tests begin. (we don't have to create for each one test)
Then I fixed it now. Sorry for taking your time.